2-Draft-Accepted/RFC00NN-PowerShellCore-Installer-Module.md

```diff
@@ -0,0 +1,78 @@
+---
+RFC: RFC00NN
+Author: Joey Aiello
+Status: Draft-Accepted
+SupercededBy: N/A
+Version: 0.1
+Area: External Module
+Comments Due: 7/15/2017
+---
+
+# PowerShellCore Installer/Utility Module
```

Comment:
Meant it as `PowerShellCore`, but I think we'll probably have to go with `PowerShellCoreProvider`.